src/main/scala/nl/knaw/dans/easy/umd/Command.scala

```diff
@@ -59,8 +59,14 @@ object Command {
     new AutoDetectReader(new FileInputStream(file))
   }
 
-  private def requireUTF8(file: File, reader: AutoDetectReader) = Try {
-    require(reader.getCharset.toString == "UTF-8", s"encoding of $file must be UTF-8 but is ${reader.getCharset}")
+  val acceptedCharsets = Set("UTF-8", "ISO-8859-1")
```

Comment:
As discussed, the command line does indeed a better job than the tika library

    > file --mime-encoding src/test/resources/*
    src/test/resources/ISO-8859-1.txt:       us-ascii
    src/test/resources/KOI8-R.txt:           iso-8859-1
    src/test/resources/ascii.txt:            us-ascii
    src/test/resources/deasy-UTF8-input.csv: utf-8
    src/test/resources/deasy-input.csv:      us-ascii
    src/test/resources/logback.xml:          us-ascii
    src/test/resources/open-office-utf8.csv: utf-8


But scala/java-wise I did not find an ideal solution so far.
* an old lengthy [discussion](http://www.scala-lang.org/old/node/1127)
* checking the [BOM](http://stackoverflow.com/questions/1288899/java-text-file-encoding)
* calling a [system process](http://kodemaniak.de/2012/11/determine-mime-type-of-a-file-in-scala/)
* _scala.io.Source.fromInputStream(inputStream)[(decoder)](scala.io.Source.fromInputStream(inputStream)(decoder))_ which can throw an error rather than apply � it would indeed flag the KOI8 file but not for all characters.